lib/trailblazer/rails/railtie.rb

```diff
@@ -3,8 +3,37 @@
 
 module Trailblazer
   class Railtie < ::Rails::Railtie
+    def self.load_root(root)
+
+      # Loader has no concept for base dir, must chdir so that it can find files local to the root
+      Dir.chdir(root) do
+        # For STI models, we need to sort all models before concepts, gather files first from loader
+        files = []
+        Loader.new.({debug: false, insert: [ModelFile, before: Loader::AddConceptFiles]}) { |file|
```

Comment:
@apotonick I could not see a way to plug this in as-is to the Loader, but I agree, that would make more sense.  

We could load _everything_ in the model directory first, not just what is hinted from the concept (which would further solve some edge cases), then just load the concepts.  

I was just trying to work with what was there.  Perhaps you can take a shot at what you would like and I'll test it out?